src/Components/Comments.jsx

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from "react";
+import { getCommentsById } from "../Utils/api";
+import CommentsCard from "./CommentsCard";
+
+const Comments = ({ article_id }) => {
+  const [comments, setComments] = useState([]);
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    setIsLoading(true);
+
+    getCommentsById(article_id).then((articleData) => {
+      setComments(articleData);
+      setIsLoading(false);
+    });
+  }, [article_id]);
+
+  return isLoading ? (
+    <section>
+      <h2 className="loading-header">Loading...</h2>
+    </section>
+  ) : (
+    <section>
+      <div className="comments-div">
+        <i className="fa-regular fa-comment"></i>{" "}
+        <p>{comments.length} comments </p>
+      </div>
+      <ul>
+        {comments.map((comment) => {
```

Comment:
You should consider what may need to change here if an article has no associated comments. How could we let the user know the section is deliberately blank, so they don't assume the comments failed to load?